StardewArchipelago/Stardew/NameMapping/NameSimplifier.cs

```diff
@@ -35,7 +35,9 @@ public string GetSimplifiedName(Item item)
                 }
                 if (name.Contains("trilobite")) // Temporary fix.
                 {
-                    displayName += " Fossil";
+                    var tempDisplayName = displayName.Split(":");
+                    tempDisplayName[1] = " Trilobite";

```

Comment:
Shouldn't you be running some kind of `displayName = displayName.Replace("something", " Trilobite");`?

This way of doing it seems overcomplicated, and risky, as you're accessing `[1]` without knowing if it exists.